avocado/core/test.py

```diff
@@ -859,7 +859,8 @@ def run_avocado(self):
             self._tag_end()
             self._report()
             self.log.info("")
-            self._stop_logging()
+            if self._config.get("run.test_runner") != 'nrunner':
+                self._stop_logging()
```

Comment:
IMO, we could drop this method. We should not keep support to others runners at this point and we don't need to introduce new code just to support the legacy runner.

Reply:
yes, I agree with you, we should drop the `_start_logging` and `_stop_logging`. I just didn't want to do it before avocado-vt [#3295](https://github.com/avocado-framework/avocado-vt/pull/3295), because last time we have compatibility issues after dropping legacy code. So I would like to wait until the compatibility between avocado and avocado-vt will be resolved.